drivers/counter/counter_mcux_rtc.c

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2018 blik GmbH
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <counter.h>
+#include <fsl_rtc.h>
+
+#define LOG_LEVEL CONFIG_COUNTER_LOG_LEVEL
+#define LOG_MODULE_NAME mcux_rtc
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
you can use recently added registeration and wrap 3 lines into 1:
`LOG_MODULE_REGISTER(mcux_rtc, CONFIG_COUNTER_LOG_LEVEL);`

Reply:
Done